app/views/energy_tariffs/energy_tariffs/choose_meters.html.erb

```diff
@@ -21,16 +21,7 @@
         <p>
           <%= t('schools.user_tariffs.choose_meters.which_meters_will_this_tariff_apply_to') %>
         </p>
-        <div class="form-check meter-choices">
-          <%= f.collection_check_boxes(:meter_ids, @meters, :id, :mpan_mprn) do |b|  %>
-            <div class="custom-control custom-checkbox">
-              <%= b.check_box(class: "custom-control-input") %>
-              <%= b.label(class: "custom-control-label spaced") do %>
-                <%= b.object.display_summary(display_data_source: false) %>
-              <% end %>
-            </div>
-          <% end %>
-        </div>
+        <%= render 'form_meter_choices', f: f %>
```

Comment:
My fault for making the card ambiguous, but rather than have a custom we should just send the user back to the standard meter choices form. As it should be possible to switch the tariff from being for a specific meter, to just all electricity/gas meters. There's a card for this so you could address there: https://trello.com/c/AYSxT3XY/3600-allow-meters-to-be-edited-on-school-tariffs

Reply:
```
  concern :tariff_holder do
    scope module: 'energy_tariffs' do
    resources :energy_tariffs do
      resources :energy_tariff_flat_prices
      resources :energy_tariff_differential_prices
      resources :energy_tariff_charges
      collection do
        get :choose_meters, to: 'energy_tariffs#choose_meters'
      end
      member do
        get :choose_type, to: 'energy_tariffs#choose_type'
        get :edit_meters, to: 'energy_tariffs#edit_meters'
        post :update_meters, to: 'energy_tariffs#update_meters'
      end
    end
  end
```

sending the user back to the standard meter choices form won't work.  `choose_meters` is a collection, not a resource (meters are selected prior to a new energy tariff being created).  so to edit the meters for an energy tariff requires a new member to be defined in the routes.